--- suphp-0.7.1/configure	2009-03-14 18:07:57.000000000 +0000
+++ suphp-0.7.1/configure	2014-07-30 16:49:54.955702538 +0100
@@ -20369,7 +20369,7 @@
                     | cut -f2 -d"/" \
                     | cut -f1 -d" "`
     major_version=`echo $APACHE_VERSION|cut -f1,2 -d.`
-    if test "$major_version" = "2.0" -o "$major_version" = "2.2"; then
+    if test "$major_version" = "2.0" -o "$major_version" = "2.2" -o "$major_version" = "2.4"; then
       APACHE_VERSION_2=true
       APACHE_VERSION_1_3=false
     else
